### PR TITLE
Update README with new config and update information

### PR DESCRIPTION
**Pull request by signalnerve**
Based on #212, but with some updates and rebase fixes. This PR:

- Updates the `config` subcommand details in the "Configuration" section to indicate that the command is no longer interactive
- Adds a section about updating `wrangler`, using Cargo and NPM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +187,20 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
...
+
+1. Updating `wrangler:
+
+    To get the latest version of Wrangler, using Cargo, run:
+
+    ```bash
+    cargo install --git https://github.com/cloudflare/wrangler --force
+    ```
+
+   To get the latest version of Wrangler, using NPM, run:
```
Comment (xtuc) [inline README.md]:
```suggestion
   To get the latest version of Wrangler, using npm, run:
```

File: README.md [hunk 2]
```diff
@@ -188,7 +187,20 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
...
+
+   ```bash
+   npm install @cloudflare/wrangler
```
Comment (xtuc) [inline README.md]:
```suggestion
   npm install -g @cloudflare/wrangler
```

do we install it globally? installing it locally will expose the executable only locally to npm at least.

**Comment by granjef3** [comment]
Can we remove references to --type=webpack supporting emscripten for now? maybe rust as well, we need a template that provides acceptable config before that "works" for most people.

File: README.md [hunk 3]
```diff
@@ -109,8 +109,7 @@ There are two types of configuration that `wrangler` uses: global user and per p
 
     To set up `wrangler` to work with your Cloudflare user, use the following commands:
 
-    - 🔧 `config`: an interactive command that asks you to pass your `email` and `api` key. Alternatively, 
-       you can use the flags `--email` and `--api-key` to the command to skip the interactive part.
+    - 🔧 `config`: a command that asks you to pass your `email` and `api` key.
```
Comment (Electroid) [inline README.md]: "a command that prompts you to enter your..." ?

**Comment by signalnerve** [comment]
> Can we remove references to --type=webpack supporting emscripten for now? maybe rust as well, we need a template that provides acceptable config before that "works" for most people.

updated!

File: README.md [hunk 4]
```diff
@@ -188,7 +187,20 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
     ```
     $ brew install openssl@1.1
     ```
-    
+
+1. Updating `wrangler:
+
+    To get the latest version of Wrangler, using Cargo, run:
+
+    ```bash
```
Comment (ashleymichal) [inline README.md]: nit
```suggestion
    ```sh
```

File: README.md [hunk 5]
```diff
@@ -188,7 +187,20 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
...
+    cargo install --git https://github.com/cloudflare/wrangler --force
+    ```
+
+   To get the latest version of Wrangler, using NPM, run:
+
+   ```bash
```
Comment (ashleymichal) [inline README.md]: nit
```suggestion
   ```sh
```

**Comment by ashleygwilliams** [comment]
merge at will!

File: README.md [hunk 6]
```diff
@@ -109,8 +109,7 @@ There are two types of configuration that `wrangler` uses: global user and per p
 
     To set up `wrangler` to work with your Cloudflare user, use the following commands:
 
-    - 🔧 `config`: an interactive command that asks you to pass your `email` and `api` key. Alternatively, 
-       you can use the flags `--email` and `--api-key` to the command to skip the interactive part.
+    - 🔧 `config`: a command that prompts you to enter your `email` and `api` key.
```
Comment (victoriabernard92) [inline README.md]: Is this true? 

```
$ wrangler config
error: The following required arguments were not provided:
    <email>
    <api-key>
```

Comment (victoriabernard92) [reply README.md]: @signalnerve

**Pull request merged**